### PR TITLE
Make upload work again

### PR DESCRIPTION
**Pull request by JanAckermann**
Upload in rich documents office overview stopped working, fixed with this change. 

Fixes:
- https://github.com/owncloud/enterprise/issues/4869#event-5770193716

### CODE REVIEW

File: templates/documents.php [hunk 1]
```diff
@@ -48,25 +48,10 @@
...
-					  id="data-upload-form"
-					  class="file_upload_form"
-					  action="<?php print_unescaped(link_to('files', 'ajax/upload.php')); ?>"
-					  method="post"
-					  enctype="multipart/form-data"
-					  target="file_upload_target_1">
-					<?php if ($_['uploadMaxFilesize'] >= 0):?>
-					<input type="hidden" name="MAX_FILE_SIZE" id="max_upload"
-						   value="<?php p($_['uploadMaxFilesize']) ?>" />
-					<?php endif;?>
```
Comment (JanAckermann) [inline templates/documents.php:61]: seems like this was never set before, only accessible while opening a file

File: templates/documents.php [hunk 2]
```diff
@@ -48,25 +48,10 @@
...
-					<!-- Send the requesttoken, this is needed for older IE versions
-						 because they don't send the CSRF token via HTTP header in this case -->
-					<input type="hidden" name="requesttoken" value="<?php p($_['requesttoken']) ?>" id="requesttoken" />
```
Comment (JanAckermann) [inline templates/documents.php:64]: Need to check with IE11

**Review by mrow4a — APPROVED** [review]

File: templates/documents.php (hunk 2 above)
Comment (JanAckermann) [reply templates/documents.php:64]: Works with IE11 not needed

**Comment by cdamken** [comment]
I tested in damken-cloud
now it works! Thanks @JanAckermann @mrow4a !!

**Comment by jnweiger** [comment]
Confirmed fixed in 2.7.0 RC2 with both classic and new Web UI.

**Pull request merged**